### DISCUSSION TITLE
How do I determine if a window exists using `ahk_exe`?

### DISCUSSION THREAD

**Discussion by trajano** [spyoungtech/ahk#321] [Q&A] [answered]
Just want to know what's the equivalent of
```
WinExists("ahk_exe ONENOTE.exe");
```

**Comment by trajano** [comment, marked as answer]
This actually works:

```python
ahk.win_exists("ahk_exe ONENOTE.EXE"):
```
I was looking for some function that would do `ahk_exe` didn't think the above would work.

**Reply by spyoungtech** [reply]
Yeah. I could see how it might be confusing since I do create individual methods for certain AHK functions, like `win_get_xxx` instead of _just_ the one `win_get` method for AHK's singular `WinGet`.

But glad you were able to figure that out :)
